Sources/NIOHTTP2/ContentLengthVerifier.swift

```diff
@@ -82,3 +92,14 @@ extension ContentLengthVerifier: CustomStringConvertible {
         return "ContentLengthVerifier(length: \(String(describing: self.expectedContentLength)))"
     }
 }
+
+extension HPACKHeaders {
+    func getPseudoHeader(name: String) -> String? {
+        for (fieldName, fieldValue, _) in self {
+            if name == fieldName {
+                return fieldValue
+            }
+        }
+        return nil
+    }
```

Comment:
I don't think we need this function: we have `headers.first(name:)` which does the same.